vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/SessionSteps.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.mobileapp.steps;
+
+import static io.appium.java_client.CommandExecutionHelper.execute;
+import static io.appium.java_client.MobileCommand.prepareArguments;
+import static org.vividus.selenium.type.CapabilitiesValueTypeAdjuster.adjustType;
+
+import java.util.AbstractMap;
+import java.util.List;
+
+import org.apache.commons.lang3.math.NumberUtils;
+import org.jbehave.core.annotations.When;
+import org.vividus.bdd.mobileapp.model.Entry;
+import org.vividus.selenium.IWebDriverProvider;
+
+import io.appium.java_client.ExecutesMethod;
+
+public class SessionSteps
```

Comment:
there is no much sense to create a new class, I believe a new step can be put to `ApplicationSteps`